Tools/Scripts/webkitpy/style/checker.py

```diff
@@ -1043,13 +1043,13 @@ def write_style_error(self,
         if self._output_format == 'vs7':
             format_string = "%s(%s):  %s  [%s] [%d]"
         else:
-            format_string = "%s:%s:  %s  [%s] [%d]"
+            format_string = "\x1b[1m%s:%s:\x1b[0m \x1b[31;1merror:\x1b[0m \x1b[33;1m[%s]\x1b[0m \x1b[1m%s\x1b[0m"
+
 
         _log.error(format_string % (file_path,
-                                           line_number,
-                                           message,
-                                           category,
-                                           confidence_in_error))
```

Comment:
It might be worth asking around if having the confidence displayed is useful to anyone before removing it!

Reply:
not only does it seem useless, it seems the vast majority didn't even know what it was!